recipes/optick/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```

- `cmake_find_package`: Will produce `Findxxxx.cmake`
- `cmake_find_package_multi`: Produces `xxxxConfig.cmake`